pkg/rtree/rtree.go

```diff
@@ -16,14 +16,16 @@ import (
 
 // Range represents a backup response.
 type Range struct {
-	StartKey []byte
-	EndKey   []byte
-	Files    []*backup.File
+	StartKey   []byte
+	EndKey     []byte
+	Files      []*backup.File
+	TotalBytes uint64
+	TotalKVs   uint64
 }
 
 // String formats a range to a string.
 func (rg *Range) String() string {
-	return fmt.Sprintf("[%x %x]", rg.StartKey, rg.EndKey)
+	return fmt.Sprintf("[%x %x %d %d]", rg.StartKey, rg.EndKey, rg.TotalKVs, rg.TotalBytes)
```

Comment:
Log like `[<start> <end> 96000 960000000]` maybe confusing, how about add some hint?
```suggestion
	return fmt.Sprintf("[%x %x %dkvs %dbytes]", rg.StartKey, rg.EndKey, rg.TotalKVs, rg.TotalBytes)
```